tests/FSharp.Test.Utilities/TestFramework.fs

```diff
@@ -10,28 +10,59 @@ open Scripting
 open Xunit
 open FSharp.Compiler.IO
 
-let inline getTestsDirectory src dir = src ++ dir
-
-// Temporary directory is TempPath + "/FSharp.Test.Utilities/" date ("yyy-MM-dd")
-// Throws exception if it Fails
-let tryCreateTemporaryDirectory () =
-    let date() = DateTime.Now.ToString("yyyy-MM-dd")
-    let now() = $"{date()}-{Guid.NewGuid().ToString()}"
-    let directory = Path.Combine(Path.GetTempPath(), now()).Replace('-', '_')
-    Directory.CreateDirectory(directory).FullName
-
-// Create a temporaryFileName -- newGuid is random --- there is no point validating the file already exists because: threading and Path.ChangeExtension() is commonly used after this API
-let tryCreateTemporaryFileName () =
-    let directory = tryCreateTemporaryDirectory ()
-    let fileName = ("Temp-" + Guid.NewGuid().ToString() + ".tmp").Replace('-', '_')
-    let filePath = Path.Combine(directory, fileName)
-    filePath
-
-// Create a temporaryFileName -- newGuid is random --- there is no point validating the file already exists because: threading and Path.ChangeExtension() is commonly used after this API
-let tryCreateTemporaryFileNameInDirectory (directory: DirectoryInfo) =
-    let fileName = ("Temp-" + Guid.NewGuid().ToString() + ".tmp").Replace('-', '_')
-    let filePath = Path.Combine(directory.FullName, fileName)
-    filePath
+let getShortId() = Guid.NewGuid().ToString().[..7]
+
```

Comment:
I found it useful to have the test artifacts in a named folder in temp. It helped with debugging this, to step through the code and easily see folders and files being created with somewhat meaningful names. Next time someone has to touch this code it will be useful again, I guess :)

Also if we want to do a quick clean up after the whole test run, we can just delete the named folder.